### ISSUE TITLE
Clicking the soft keyboard with a mouse does not enter any characters with GV

### ISSUE THREAD

**Issue by cpeterso** [mozilla-mobile/firefox-tv#1902]
## Steps to reproduce

@vickychin says clicking soft keyboard with a mouse doesn’t work with FFTV+GV.

### Expected behavior

Typing on external keyboard works.

### Actual behavior

Clicking the soft keyboard with a mouse does not enter any characters.

### Device information
* Fire TV device: Vicky saw this problem on the Cube, but she hasn't tested whether this bug happens on other Fire TV devices.

**Comment by liuche** [comment]
We took a survey where 98% of the respondents use a Fire TV remote, so we're labeling a P3.

Vicky, @cpeterso is this blocking GV development work? Otherwise we don't think this is a common use case for our users.

**Comment by cpeterso** [comment]
The GV team is not currently blocked by this issue. It is mostly just an annoyance for testing. We don't know yet if this is a GV or FFTV front end bug, so someone needs to investigate.

**Comment by cpeterso** [comment]
I don't know if this is a FFTV or GV bug, but I filed a new GV issue to track this problem: https://bugzilla.mozilla.org/show_bug.cgi?id=1536864

**Comment by athomasmoz** [comment]
Thanks, I'm going to close as won't fix, so that we can focus on the other issues.

**Comment by makotokato** [comment]
This occurs on system WebView build, not GV.  I guess that this issue may be FireTV keyboard issue.

**Comment by cpeterso** [comment]
> This occurs on system WebView build, not GV. I guess that this issue may be FireTV keyboard issue.

Does Amazon's software keyboard work correctly in the Silk browser? Does FFTV receive any key events at all from Amazon's software keyboard?

**Comment by makotokato** [comment]
> Does Amazon's software keyboard work correctly in the Silk browser? Does FFTV receive any key events at all from Amazon's software keyboard?

No, even using Silk browser with builtin software keyboard, this issue occurs.  Software keyboard doesn't support mouse operation.
